tool/filter_missed_deposits/filter_missed_btc.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+zetatool filterdeposit btc --config ./tool/filter_missed_deposits/InboundTxFilter_config.json
```

Comment:
Why created sub scripts instead of directly putting the command in the makefile?